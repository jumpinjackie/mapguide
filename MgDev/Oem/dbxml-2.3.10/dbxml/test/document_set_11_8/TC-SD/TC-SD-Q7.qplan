<XQuery>
  <FLWOR>
    <ForBinding name="word">
      <DbXmlNav>
        <QueryPlanFunction result="collection" container="11.8.4.dbxml">
          <OQPlan>V(node-element-equality-string,qd,=,'1900')</OQPlan>
        </QueryPlanFunction>
        <DbXmlStep axis="child" name="dictionary" nodeType="element"/>
        <DbXmlStep axis="child" name="e" nodeType="element"/>
      </DbXmlNav>
      <Where>
        <EveryFLWOR>
          <ForBinding name="item">
            <DocumentOrder>
              <DbXmlNav>
                <Variable name="word"/>
                <DbXmlStep axis="child" name="ss" nodeType="element"/>
                <DbXmlStep axis="child" name="s" nodeType="element"/>
                <DbXmlStep axis="child" name="qp" nodeType="element"/>
                <DbXmlStep axis="child" name="q" nodeType="element"/>
              </DbXmlNav>
            </DocumentOrder>
          </ForBinding>
          <DbXmlNav>
            <Variable name="item"/>
            <DbXmlCompare name="equal" join="child" name="qd" nodeType="element">
              <Sequence>
                <AnyAtomicTypeConstructor value="1900" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
              </Sequence>
            </DbXmlCompare>
          </DbXmlNav>
        </EveryFLWOR>
      </Where>
    </ForBinding>
    <Variable name="word"/>
  </FLWOR>
</XQuery>
