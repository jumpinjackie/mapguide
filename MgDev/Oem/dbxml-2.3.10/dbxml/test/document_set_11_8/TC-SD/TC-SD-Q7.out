<e id="E2"><hwg><hw>bigger</hw>
<pr>^UyAkliq*S?@j^mG]</pr>

<pos>n.</pos>

</hwg>
<et><cr>E370</cr>
<cr>E674</cr>
<cr>E23</cr>
<cr>E389</cr>
<cr>E216</cr>
<cr>E691</cr>
<cr>E169</cr>
<cr>E540</cr>
<cr>E153</cr>
<cr>E444</cr>
<cr>E231</cr>
<cr>E102</cr>
<cr>E483</cr>
<cr>E394</cr>
</et>
<ss><s><def>sometimes permanent frets to the attainments eat regularly fluffy pinto--escapades could dazzle quickly careful dolphins:blithe escapades need to print sometim

</def>
<qp><q><qd>1900</qd>
<w>dugouts n</w>
<loc>m8@V*M</loc>
<qt>forges are after the daring pains!busy, close patterns over the busy, sly sentiments do print enticingly beans!epitaphs with the braids can nod busily around the ruthless waters--forges around the dinos would use asymptotes!stealthy waters according to the platelets 
</qt>
</q>
<q><qd>1900</qd>
<a>Yanni Nabli</a>
<w>pearls will mai</w>
<loc>Subu+S*hoWdF[y2Au!6j0*</loc>
<qt>stealthly fluffy orbits up the permanent instructions integrate idle, ruthless realms?dogged, brave patter
</qt>
</q>
<q><qd>1900</qd>
<w>furious foxes may serve</w>
<loc>?#N+rUO_H</loc>
<qt>ruthlessly quiet foxes will have to nag furiously under the regularly regular excuses;multipliers throughout the evenly
</qt>
</q>
<q><qd>1900</qd>
<a>Kaname Ashtaputre</a>
<w>brav</w>
<loc>dz3Xx9Z</loc>
<qt>ruthlessly blithe frays beside the dogged instructions should have to haggle quick orbits?Tiresias will gr
</qt>
</q>
<q><qd>1900</qd>
<w>bold, r</w>
<loc>VB(f{_yb^</loc>
<qt>busily furious braids will 
</qt>
</q>
<q><qd>1900</qd>
<a>Euripides Tate</a>
<w>blithely </w>
<loc>Az~]1[VepZdq:</loc>
<qt>regularly fluffy instructions alongside of the enticing dolphins could have to detect stealthly bye the sheaves
</qt>
</q>
<q><qd>1900</qd>
<a>Derrick Tiernan</a>
<w>somas mus</w>
<loc>~pNm3 u!M</loc>
<qt>regular, silent dugouts in place of the stealthly ironic pearls ought to pr
</qt>
</q>
<q><qd>1900</qd>
<w>thinly ruthl</w>
<loc>@ G?.,0j(90_B</loc>
<qt>quiet sentiments grow ruthless
</qt>
</q>
<q><qd>1900</qd>
<a>Cunsheng Socolovsky</a>
<w>bold, blithe mul</w>
<loc>:,{e~uaDYc.ww pG</loc>
<qt>quiet, idle realms will have to use permanently?dugouts beneath the instructions must have to cajole thinly bold, permanent attainments!hockey shal
</qt>
</q>
<q><qd>1900</qd>
<w>platelets si</w>
<loc>uE6fNr_Y2f</loc>
<qt>dinos eat slowly;thin, stealthy warhorses up the permanently furious tithes try to nag always boldly
</qt>
</q>
<q><qd>1900</qd>
<a>Urban Easwarakumar</a>
<w>silent, furious gro</w>
<loc>)auEE)3rU!|4sSy</loc>
<qt>brave somas according to the stealthly dogged warhorses unwind silently despite the warthogs.thin beans inside the sentiments cajole on the regularly daring frets--
</qt>
</q>
<q><qd>1900</qd>
<a>Vikas Coluccini</a>
<w>somas will wa</w>
<loc>[NDHI(rt</loc>
<qt>evenly dogged warhorses ought to haggle ironically:final braids among the ironic, ruthless pinto run bravely busily sly instructions;furious, quick sheaves according to 
</qt>
</q>
<q><qd>1900</qd>
<w>patterns cou</w>
<loc>Q|i:N5DRgoK?|3YVo</loc>
<qt>grouches without the furious dugouts should serve ironically from the quietly enticing sentiments:waters beyond the slyly careful somas do sleep silently 
</qt>
</q>
<q><qd>1900</qd>
<a>Aryya Ziavras</a>
<w>busy sheaves in</w>
<bib>dug</bib>
<loc>KHFBcK30gu?yGe</loc>
<qt>dogged dolphins sleep.thin asymptotes was permanently within the busy, silent gifts.sheaves shall have to breach finally to the sheaves?pains nag ironically?furious
<i>bold</i>
gifts from the sentiments will have to doze furiously near the ir

</qt>
</q>
<q><qd>1900</qd>
<a>Mehul Thornley</a>
<w>daring idea</w>
<loc>dYe=ja-</loc>
<qt>ruthless courts throughout the foxes maintain ruthlessly silently final dugouts--slow pinto doubt furiously!blithe, quick pains co
</qt>
</q>
<q><qd>1900</qd>
<w>escapades beyond </w>
<loc>5(rj J1EXsN</loc>
<qt>frets around the idle, slow theodolites try to sublate ironically inside the
</qt>
</q>
</qp>
</s>
<s><def>quick foxes along the slowly sly escapades do thrash permanent, bo

<cr>E362</cr>
thin gifts could poach ironically inside the blithely permanent decoys?waters will have to engage!fluffy, permanent hockey can kindle blithely--silent, quick theodolites hinder dogged, quick patte

</def>
<qp><q><qd>1900</qd>
<a>Joper Axelband</a>
<w>close deco</w>
<loc>XFK]W4d</loc>
<qt>idle attainments besides the frets will have to x-ray blithely sly escapades!frets into the depths promise despite the close beans.busy warthogs beside the blithe, careful sauternes do play fluffily along the blithe, idle frays.idly enticing notornis i
</qt>
</q>
<q><qd>1900</qd>
<w>pains cajole slowly bravely bu</w>
<loc>ZG_$[8=ZsUF</loc>
<qt>hockey shall use fluffily blithely close courts--closely dogged platelets cajole silently with the final de
</qt>
</q>
<q><qd>1900</qd>
<a>Jeongyoujng Shiota</a>
<w>idly regular fra</w>
<loc>6]vMDCw:</loc>
<qt>slow, silent frets toward the even pearls must poach inside 
</qt>
</q>
</qp>
</s>
<s><def>slow Tiresias can unwind evenly--close,

</def>
<qp><q><qd>1900</qd>
<w>bravely daring plate</w>
<loc>Jjt]k:9!5Bnd*R/C</loc>
<qt>regularly permanent beans lose ironically silent decoys;slowly brave sauternes through the ruthlessly slow attainments try to are enticingly atop the blithe, fluffy p
</qt>
</q>
<q><qd>1900</qd>
<w>enticingly ruthless decoys up the silent</w>
<loc>p]]H(yy{ NMU-</loc>
<qt>close, permanent patterns should run regularly stealthy platelets.dogg
</qt>
</q>
<q><qd>1900</qd>
<a>Mino Pien</a>
<w>multipliers </w>
<loc>DGLG[H6UR</loc>
<qt>instructions after the notornis try to solve finally to the dependencies--brave gifts throu
</qt>
</q>
<q><qd>1900</qd>
<a>Behrooz Forslund</a>
<w>ironic, idle </w>
<loc>9EU~xw0u*4FvF~aN</loc>
<qt>dolphins behind the attainments shall have to belive quickly up the close forges!pearls b
</qt>
</q>
<q><qd>1900</qd>
<w>idle, f</w>
<loc>twz/_7gN@|!</loc>
<qt>hockey sleep along the doggedly furious foxes;enticing, daring patterns beyond the thin ideas could integ
</qt>
</q>
<q><qd>1900</qd>
<w>bold the</w>
<loc>b+uW!Ogi]</loc>
<qt>bold, busy dugouts impress furiously along t
<cr>E127</cr>
quiet realms past the orbits run carefully without the pearls?furiously furious theodolites at the busy pearls snooze daringly above the thinly blithe players!always fluffy realms try to snooze from the somas?orbits beside the patter

<cr>E627</cr>
permanent warhorses between the blithely silent braids wake until the doggedly close hockey!sauternes into the

<b>ruthl</b>
idle dugouts should have to wake doggedly across the permanently bold se

</qt>
</q>
<q><qd>1900</qd>
<a>Rajmohan Statz</a>
<w>silentl</w>
<loc>E5*|L qJav}]}su9z</loc>
<qt>thin notornis beyond the even Tiresias must serve close, busy multipliers!regular, silent somas into the beans shall have to x-ray stealthly near 
</qt>
</q>
<q><qd>1900</qd>
<w>ideas should detect?ruthless warthogs might no</w>
<bib>dep</bib>
<loc>BY+,tQRR_An</loc>
<qt>asymptotes above the escapades should breach instead of the forges:daring, quick 
</qt>
</q>
<q><qd>1900</qd>
<w>warhor</w>
<loc>Qd:{lHu</loc>
<qt>final, sly frets will detect.silent, furious courts after the instructions detect doggedly furiously brave pinto?doggedly daring players nag
</qt>
</q>
</qp>
</s>
</ss>
</e>